src/com/github/TylerWhitmore/cs3230/AssignmentTwo.java

```diff
@@ -0,0 +1,139 @@
+package src.com.github.TylerWhitmore.cs3230;
```

Comment:
You'll actually remove the `src` from the package declaration. Leave the folder though.